packages/web-vite/src/components/HighlightSidebar.tsx

```
@@ -0,0 +1,266 @@
+            {/* Color indicator bar */}
+            <div
+              className={`highlight-color-bar highlight-bar-${highlight.color}`}
+            />
+
+            <div className="highlight-body">
+              {/* Quote */}
+              <div
+                className="highlight-quote"
+                onClick={() => onJumpToHighlight(highlight.id)}
+                role="button"
+                tabIndex={0}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)